### PR TITLE
Revisions throughout manuscript and response to reviewers

### PR DESCRIPTION
**Pull request by naupaka**
Caught a number of proofreading errors and clarified meaning in a number of spots. Added citation for _Morpho_.

### CODE REVIEW

**Comment by fmichonneau** [comment]
I read over the changes and it looks good to me. Thanks for doing this Naupaka!

I was thinking we could also add a sentence in the introduction to mention the glossary.

**Comment by emhart** [comment]
I can hold off on the merge if we want to do that.

File: manuscript/manuscript.md [hunk 1]
```diff
@@ -76,98 +76,122 @@ organization:
...
+data, a repository such as those run by the National Center for
+Biotechnology Information (NCBI) (e.g. GenBank) provides not only a
+location for data archival, but also encourages a set of practices
+related to consistent data formatting and the inclusion of appropriate
+metadata. However, data storage policies are highly variable between
+repositories [@Marcial2010]. A data management plan utilizing best
+practices across all stages of the data life cycle will facilitate
+transition from local storage to repository [@Michener2015].
+Similarly, having such a plan can facilitate transition from
+repository to repository if funding runs out or needs change. Good
```
Comment (snim2) [inline manuscript/manuscript.md]: This sentence confused me a little on first reading. Can we change "needs" to "requirements"?

File: manuscript/manuscript.md [hunk 2]
```diff
@@ -178,334 +202,376 @@ None of these questions have universal answers, nor are they the only
...
+validity of statistical models, or to directly compare findings across
+studies.
+
+Therefore, data should always be kept in raw format whenever possible
+(within the constraints of technical limitations). In addition to
+being the most appropriate way to ensure transparency in analysis,
+having the data stored and archived in their original state gives a
+common point of reference for derivative analyses. What constitutes
+sufficiently "raw" data is not always clear (e.g., ohms off a
+temperature sensor or images off an Illumina sequencing flowcell are
```
Comment (snim2) [inline manuscript/manuscript.md]: "from a" rather than "off a"

File: manuscript/manuscript.md [hunk 3]
```diff
@@ -178,334 +202,376 @@ None of these questions have universal answers, nor are they the only
...
-loss due to theft or natural disasters.
+Every storage medium can fail, and every failure can result in loss of
+data. Researchers should therefore back data up at all stages of the
+research process. Data stored on local computers or institutional
+servers during the collection and analysis phases of a project should
+be backed up to other locations to protect against data loss. No
+backup system is failsafe (see the stories of the
+[Dedoose crash][dedoose] and the [near deletion of Toy Story 2][ts2]),
+so more than one backup system should be used. Ideally you should have
+two onsite copies (such as on a computer, an external hard drive, or a
```
Comment (snim2) [inline manuscript/manuscript.md]: Are **onsite** and **offsite** hyphenates?

**Comment by snim2** [comment]
This looks great to me, many thanks @naupaka

**Comment by naupaka** [comment]
I will add changes suggested by @fmichonneau and @snim2. Wait to merge...

File: manuscript/manuscript.md (hunk 1 above)
Comment (naupaka) [inline manuscript/manuscript.md]: changed

File: manuscript/manuscript.md (hunk 2 above)
Comment (naupaka) [inline manuscript/manuscript.md]: changed

File: manuscript/manuscript.md (hunk 3 above)
Comment (naupaka) [inline manuscript/manuscript.md]: yes, they are. changed.

**Comment by naupaka** [comment]
Ok. Changes made.

**Pull request merged**